facia-press/app/AppLoader.scala

```diff
@@ -19,25 +20,18 @@ class AppLoader extends FrontendApplicationLoader {
   override def buildComponents(context: Context): FrontendComponents = new BuiltInComponentsFromContext(context) with AppComponents
 }
 
-trait Controllers {
-  def wsClient: WSClient
-  def toolPressQueueWorker: ToolPressQueueWorker
-  def liveFapiFrontPress: LiveFapiFrontPress
-  def draftFapiFrontPress: DraftFapiFrontPress
-  lazy val healthCheck = wire[HealthCheck]
-  lazy val applicationController: Application = wire[Application]
-}
+trait AppComponents extends FrontendComponents {
```

Comment:
I tried the approach of one single big `AppComponents` (not getting rid of `FrontendComponents` obviously) we talked about a few days ago. It works quite well for `facia-press` since it is not this big and dev-build/preview don't depend on it.
Let me know what you think. (Note: I have done the same thing with the Rss app for the same reason. See below)

I will try it on `article` or `applications` at some point in the future, that's where trouble will appear I think. 😉